airbyte-integrations/connectors/source-recharge/integration_tests/abnormal_state.json

```diff
@@ -0,0 +1,21 @@
+{
+  "addresses": {
+    "created_at": "2024-05-18T00:00:00"
```

Comment:
I assume you are not going to be around at that time?)

Reply:
Set other dates